### PR TITLE
feat: Making extensions accessible without having to add additional using statements

### PR DESCRIPTION
**Pull request by nickrandolph**
## PR Type

What kind of change does this PR introduce?
- Refactoring (no functional changes, no api changes)

## What is the current behavior?

For each extension (eg Navigation) in order to add to HostBuilder in App.xaml.host.cs the corresponding namespace needs to be imported (ie Uno.Extensions.Navigation).

## What is the new behavior?

HostBuilder extensions are now in the Uno.Extensions namespace, which will make it easier to use each extension without having to add using statements

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](doc/articles/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: testing/TestHarness/TestHarness.Shared/GlobalUsings.cs [hunk 1]
```diff
@@ -47,21 +35,22 @@
...
+	global using Windows.UI.Xaml.Controls.Primitives;
+	global using Windows.UI.Xaml.Input;
+	global using Windows.UI.Xaml.Navigation;
+	global using Windows.UI.Xaml.Markup;
+	global using Windows.UI.Xaml.Data;
+	global using Windows.UI.Xaml.Media;
+	global using Window = Windows.UI.Xaml.Window;
+	global using NavigationView = Microsoft.UI.Xaml.Controls.NavigationView;
+	global using NavigationViewSelectionChangedEventArgs = Microsoft.UI.Xaml.Controls.NavigationViewSelectionChangedEventArgs;
+	global using LaunchActivatedEventArgs = Windows.ApplicationModel.Activation.LaunchActivatedEventArgs;
```
Comment (Xiaoy312) [inline testing/TestHarness/TestHarness.Shared/GlobalUsings.cs:53]: this is overzealous; it is probably best to keep that using for app.xanml.cs

File: testing/TestHarness/TestHarness.Shared/TestHarness.Shared.shproj [hunk 2]
```diff
@@ -10,5 +10,7 @@
   <PropertyGroup />
   <Import Project="TestHarness.Shared.projitems" Label="Shared" />
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.CSharp.targets" />
-  
+  <ItemGroup>
+    <_Globbed_Compile Remove="_Compat\IsExternalInit.cs" />
+  </ItemGroup>
```
Comment (Xiaoy312) [inline testing/TestHarness/TestHarness.Shared/TestHarness.Shared.shproj:15]: remove

File: testing/TestHarness/TestHarness.Shared/_Compat/IsExternalInit.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Linq;
```
Comment (Xiaoy312) [inline testing/TestHarness/TestHarness.Shared/_Compat/IsExternalInit.cs:2]: arent those in the global-using?

**Review by Xiaoy312 — APPROVED** [review]

File: testing/TestHarness/TestHarness.Shared/GlobalUsings.cs (hunk 1 above)
Comment (nickrandolph) [reply testing/TestHarness/TestHarness.Shared/GlobalUsings.cs]: Window and LaunchActivatedEventArgs moved to app.xaml.cs. The other two are used in multiple places so have been left here

File: testing/TestHarness/TestHarness.Shared/_Compat/IsExternalInit.cs (hunk 3 above)
Comment (nickrandolph) [reply testing/TestHarness/TestHarness.Shared/_Compat/IsExternalInit.cs]: removed

File: testing/TestHarness/TestHarness.Shared/TestHarness.Shared.shproj (hunk 2 above)
Comment (nickrandolph) [reply testing/TestHarness/TestHarness.Shared/TestHarness.Shared.shproj]: removed

**Pull request merged**